faq/obo-file-format-support.md

```diff
@@ -0,0 +1,62 @@
+
+
+
+# Support for OBO File Format
+A number of ontologist develop their ontologies using the [OBO file format](https://owlcollab.github.io/oboformat/doc/GO.format.obo-1_4.html). This page provides support for maintaining OBO file ontologies in accordance with OBO Foundry principles.   
+The page is divided in sections containing OBO specific support for that principle.
+
+## Principle 1: Open
+
+OBO Foundry ontologies MUST specify the reuse constraints using the following annotations in any publically released OWL version of the ontology:
```

Comment:
OBO Foundry ontologies MUST specify the reuse constraints: add link to these.
Then make the OWL annotations a sub-point, like the OBO one.